main.go

```diff
@@ -0,0 +1,210 @@
+	"github.com/statechannels/go-nitro/client"
+	"github.com/statechannels/go-nitro/client/engine"
+	"github.com/statechannels/go-nitro/client/engine/chainservice"
+	NitroAdjudicator "github.com/statechannels/go-nitro/client/engine/chainservice/adjudicator"
+	Create2Deployer "github.com/statechannels/go-nitro/client/engine/chainservice/create2deployer"
+	p2pms "github.com/statechannels/go-nitro/client/engine/messageservice/p2p-message-service"
+	"github.com/statechannels/go-nitro/client/engine/store"
+	"github.com/statechannels/go-nitro/crypto"
+	"github.com/statechannels/go-nitro/rpc"
+	"github.com/statechannels/go-nitro/rpc/transport"
+	"github.com/statechannels/go-nitro/rpc/transport/nats"
+	"github.com/statechannels/go-nitro/rpc/transport/ws"
+	"github.com/statechannels/go-nitro/types"
+	"github.com/tidwall/buntdb"
+)
+
+func main() {
+	var pkString, hardhatUrl string
+	var msgPort, rpcPort int
+	var useNats, useDurableStore bool
+
+	flag.BoolVar(&useNats, "useNats", true, "Specifies whether to use NATS or http/ws for the rpc server.")
+	flag.BoolVar(&useDurableStore, "useDurableStore", false, "Specifies whether to use a durable store or an in-memory store.")
+	flag.StringVar(&pkString, "pk", "2d999770f7b5d49b694080f987b82bbc9fc9ac2b4dcc10b0f8aba7d700f69c6d", "Specifies the private key for the client. Default is Alice's private key.")
+	flag.StringVar(&hardhatUrl, "hardhatUrl", "ws://127.0.0.1:8545", "Specifies the url for the hardhat node.")
+	flag.IntVar(&msgPort, "msgPort", 3005, "Specifies the tcp port for the  message service.")
+	flag.IntVar(&rpcPort, "rpcPort", 4005, "Specifies the tcp port for the rpc server.")
+
+	flag.Parse()
+
+	pk := common.Hex2Bytes(pkString)
+	me := crypto.GetAddressFromSecretKeyBytes(pk)
+
+	logDestination := os.Stdout
+
+	var ourStore store.Store
+	if useDurableStore {
+		dataFolder := fmt.Sprintf("./data/nitro-service/%s", me.String())
+		ourStore = store.NewDurableStore(pk, dataFolder, buntdb.Config{})
+	} else {
+		ourStore = store.NewMemStore(pk)
+	}
+
+	chainService, err := newChainService(context.Background(), *ourStore.GetAddress(), hardhatUrl)
+	if err != nil {
+		panic(err)
+	}
+
+	messageservice := p2pms.NewMessageService("127.0.0.1", msgPort, *ourStore.GetAddress(), pk)
+
+	node := client.New(
+		messageservice,
+		chainService,
+		ourStore,
+		logDestination,
+		&engine.PermissivePolicy{},
+		nil)
+
+	var transport transport.Responder
+
+	if useNats {
+		transport, err = nats.NewNatsTransportAsServer(rpcPort)
+	} else {
+		transport, err = ws.NewWebSocketTransportAsServer(fmt.Sprint(rpcPort))
+	}
+	if err != nil {
+		panic(err)
+	}
+
+	logger := zerolog.New(logDestination).
+		Level(zerolog.TraceLevel).
+		With().
+		Timestamp().
+		Str("client", ourStore.GetAddress().String()).
+		Str("rpc", "server").
+		Str("scope", "").
+		Logger()
+	_, err = rpc.NewRpcServer(&node, &logger, transport)
+	if err != nil {
+		panic(err)
+	}
+
+	fmt.Println("Nitro as a Service listening on localhost:", rpcPort)
```

Comment:
less annoying than, eg, `localhost: 3452`

```suggestion
	fmt.Println("Nitro as a Service listening on port", rpcPort)
```